pkg/model/initializers.go

```diff
@@ -54,6 +56,7 @@ var backends []string = []string{
 	ReplitBackend,
 	StarcoderBackend,
 	BloomzBackend,
+	LCHuggingFaceBackend,
```

Comment:
as this backend requires in any case explicit configuration, I wouldn't add it here as it will attempt to autoloads the backends

Reply:
It will work if backend not in list?

Reply:
yes